native/src/components/CalendarRangeModal.tsx

```diff
@@ -20,7 +20,7 @@ const DatePickerWrapper = styled.View`
 
 const StyledView = styled.View`
   gap: 8px;
-  justify-content: ${props => (props.theme.contentDirection === 'rtl' ? 'flex-start' : 'flex-end')};
+  justify-content: flex-end;
```

Comment:
Not only do we not have the `contentDirection` prop in the `native` part, RN is also smart enough to already flip large parts of the displays in rtl languages.

Reply:
Yep .. no need for contentDirection here.